### PR TITLE
Brand soda quest dialogue

### PR DESCRIPTION
**Pull request by funnyboy044**
Feedback on added dialogue would be cool.
If you want to add any jokes of your own, feel free!

### CODE REVIEW

File: game/money.rpy [hunk 1]
```diff
@@ -33,8 +33,15 @@ label dolly:
 
 label .dbs:
     show posty neutral
-    show bs follow behind posty
-    p "_" #todo: #111 Dolly quickly rejects Brand Soda, saying they've already spoken.
+    p "I don't think this'll work, but let's try it anyway." #todo: #111 Dolly quickly rejects Brand Soda, saying they've already spoken.
```
Comment (Martysh12) [inline game/money.rpy:36]: Remove this TODO comment

File: game/toasty_hints.rpy [hunk 2]
```diff
@@ -159,7 +169,18 @@ label toasty_hints:
...
-        t "hint for if you finished Brand Soda quest" # TODO: #18 hint for after brand soda quest (go in the museum)
+        t "You deal with that Brand Soda guy yet?"
+        p "Yeah. Yellow Diamond's gonna be their manager, to some degree."
+        t "Oh good. Now what?"
+        p "I'm not sure. I got some money out of it."
+        t smug2 "Try spending it all in one place."
+        p annoyed "Very funny."
+        t "Ok, seriously though, try going somewhere else. You spent so much time wandering around the street with Brand Soda."
+        t "Maybe you should go to that art museum. It sounds interesting!"
+        t "You could the next big painting and get a fat paycheck from it!"
```
Comment (Martysh12) [inline game/toasty_hints.rpy:180]: > You could the next big painting

File: game/room001-mainstreet.rpy [hunk 3]
```diff
@@ -563,8 +672,9 @@ label .tooly:
 
     if party_bs:
         show posty neutral
-        show bs follow behind posty
-        p "_" #275 talking to tooly with brand soda
+        show bs follow behind posty with moveinleft
+        bs "Excuse me sir! Would you like to-"
+        tooly "I don't deal with all these newfandangled social experiments! Beat it!"
```
Comment (Martysh12) [inline game/room001-mainstreet.rpy:677]: Is "newfandangled" intentional?

File: game/room001-mainstreet.rpy [hunk 4]
```diff
@@ -537,8 +635,19 @@ label .btnet:
...
+        p "We're looking for someone to market them."
+        btnet "Aside from maybe a job at the museum, I'm afraid there's nothing I can do."
+        p "Maybe you could give an object-on-the-street interview about them?"
+        bs "Yeah, old man, give your honest thoughts!"
+        btnet "Ummmmmm..."
+        btnet "They make bad first impressions, expect rewards too quickly, has delusions of grandeur..."
+        btnet "Smells flat..."
+        bs "We're leaving."
+        p "Talk to you later!"
+         # todo: #109
```
Comment (Martysh12) [inline game/room001-mainstreet.rpy:650]: Remove this TODO comment

File: game/room001-mainstreet.rpy [hunk 5]
```diff
@@ -447,23 +472,81 @@ label .yd:
     elif quest.bs:
         jump .yd_bs_happy
     else:
-        p "_" # TODO: #15 conversation where yd implies they want to get into advertising
-        yd "_"
+        yd "HIIIIIIIIIII!"
```
Comment (Martysh12) [inline game/room001-mainstreet.rpy:475]: Tone down the caps maybe?

File: game/room001-mainstreet.rpy [hunk 6]
```diff
@@ -447,23 +472,81 @@ label .yd:
...
 
 label .yd_bs_money:
-    show bs follow behind posty
-    p "_" # TODO: #16 conversation where posty convinces yd to support Brand Soda. Brand Soda, grateful, gives Posty some money as thanks.
-    yd "_"
-    bs "_"
+    show bs follow behind posty with moveinleft
+    yd "Hi Maily!"
+    p "Posty."
+    yd "Hi Posty! Hi Soday!"
```
Comment (Martysh12) [inline game/room001-mainstreet.rpy:507]: "Soday"?

File: game/room001-mainstreet.rpy [hunk 7]
```diff
@@ -473,16 +556,23 @@ label .yd_bs_money:
     hide cash_bundle_1
     call money_get
 
-    bs "_"
+    p happy "Wow, thanks Brand Soda!"
+    bs "I'll never forget you!"
+    bs "So kid, what is this buisness oppurtunity you were starting again?"
```
Comment (Martysh12) [inline game/room001-mainstreet.rpy:561]: I suggest putting double quotes around "buisness oppurtunity"

**Comment by funnyboy044** [comment]
feedback on the length of the dialogue and emotions would also be appreciated

**Review by pixequil — APPROVED** [review]

**Pull request merged**